samples/tutorial/sym1/symTiny.c

```diff
@@ -0,0 +1,42 @@
+// --------------------------------------------------------------------------
```

Comment:
We don't want any new programs in the tutorial. Please remove it or move it to `targettest/sym1`.

Reply:
**FIXED**   Please check and confirm.